cinema/serializers.py

```diff
@@ -51,13 +67,23 @@ class MovieDetailSerializer(MovieSerializer):
 
     class Meta:
```

Comment:
You are inheriting MovieListSerializer from MovieSerializer, where you already defined class Meta, so you can delete it from MovieListSerializer.